manifests/rule.pp

```diff
@@ -35,11 +35,15 @@
   }
   $saddr_real = $saddr ? {
     undef   => '',
-    default => "saddr @ipfilter(${saddr})",
+    Array   => "saddr @ipfilter(join(${daddr}, ', '))",
```

Comment:
you mixed saddr/daddr here:
```suggestion
    Array   => "saddr @ipfilter(join(${saddr}, ', '))",
```

Reply:
Fixed in 6af0e676910ef1355682188edeabceccbf1d3372

Reply:
It seems like you didn't push the commit?

Reply:
The reference is broken as I force pushed changes to that commit. New id is 64d0f763ec22c566fc59d371f0e2d4c0be05140f